nengo_gui/static/lib/js/interact-1.2.4.js

```diff
@@ -3354,6 +3354,7 @@
         this.shiftKey  = event.shiftKey;
         this.metaKey   = event.metaKey;
         this.button    = event.button;
+        this.buttons   = event.buttons;
```

Comment:
I thought about this and decided to open a separate PR. In case there will be any problems with the new interactjs version this can be tested independent from the changes in this PR.